cg/server/invoices/templates/invoices/new.html

```diff
@@ -100,7 +100,7 @@ <h1>Make new invoice ({{record_type}})</h1>
   function check_subset() {
     total_price = 0;
     $("input[name='records']").each(function() {
-      if (total_price <= 500000) {
+      if (total_price + parseInt(this.id) <= 750000) {
```

Comment:
If the threshold needs to be updated once in a while, should it be a config-setting?

Reply:
sure. You want to have a config here: cg/server/invoices/config.py?

Reply:
No I think I'd rather want it as either a property on customer if it varies with customer or if it is not varying with customer having it in the ordinary config yaml for cg under a new section invoicing: 

Does it make sense to you?

Reply:
I added it to the server config instead. Seemed to make more sense.